contrib/action_nitro/Makefile

```diff
@@ -1,3 +1,6 @@
+.SUFFIXES:
```

Comment:
According to <https://www.gnu.org/prep/standards/html_node/Makefile-Basics.html>, all Makefiles should define `.SUFFIXES` and `SHELL` to avoid surprises. Some setups may define built-in rules that cause unexpected behaviours. I prefer to be systematic than to work on a case-by-case basis.